quantum/action.c

```diff
@@ -497,7 +497,7 @@ void process_action(keyrecord_t *record, action_t action) {
                 default:
                     if (event.pressed) {
                         if (tap_count > 0) {
-#    ifdef HOLD_ON_OTHER_KEY_PRESS_PER_KEY
+#    ifdef HOLD_ON_OTHER_KEY_PRESS
```

Comment:
Life got crazy but I finally got around to looking at this again after stale-bot reminded me. Should be good to go again, other combo tests are failing right now so the action won't run but they're passing.

The issues weren't actually from that PR, they were from a couple of the things I had thrown in in the last PR to make Retro Shift ready for `HOLD_ON_OTHER_KEY_PRESS` before it happened not being quite correct. I also fixed the above, which I think wasn't noticed in the three months since the switch because an earlier evaluation was also added (which I had to exclude Retro Shift from).